To: vim_dev@googlegroups.com
Subject: Patch 7.3.208
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.208
Problem:    Early terminated if statement.
Solution:   Remove the semicolon. (Lech Lorens)
Files:      src/gui_mac.c


*** ../mercurial/vim73/src/gui_mac.c	2011-01-17 20:08:04.000000000 +0100
--- src/gui_mac.c	2011-06-06 05:35:46.000000000 +0200
***************
*** 1840,1846 ****
  	p.v -= gui.scrollbar_height;
      p.v -= p.v % gui.char_height;
      p.v += 2 * gui.border_width;
!     if (gui.which_scrollbars[SBAR_BOTTOM]);
  	p.v += gui.scrollbar_height;
  
      ZoomWindowIdeal(whichWindow, thePart, &p);
--- 1840,1846 ----
  	p.v -= gui.scrollbar_height;
      p.v -= p.v % gui.char_height;
      p.v += 2 * gui.border_width;
!     if (gui.which_scrollbars[SBAR_BOTTOM])
  	p.v += gui.scrollbar_height;
  
      ZoomWindowIdeal(whichWindow, thePart, &p);
*** ../vim-7.3.207/src/version.c	2011-06-12 20:31:25.000000000 +0200
--- src/version.c	2011-06-12 20:33:00.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     208,
  /**/

-- 
Keep America beautiful.  Swallow your beer cans.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
